openassessment/xblock/static/js/spec/lms/oa_response.js

```diff
@@ -5,6 +5,23 @@ Tests for OpenAssessment response (submission) view.
 describe("OpenAssessment.ResponseView", function() {
 
     var FAKE_URL = "http://www.example.com";
+    var UPLOADABLE_IMAGE_MIME_TYPES = [
+        'image/gif',
+        'image/jpeg',
+        'image/pjpeg',
+        'image/png',
+    ];
+
+    var UPLOADABLE_FILE_MIME_TYPES = [
+        'application/pdf',
+        'image/gif',
+        'image/jpeg',
+        'image/pjpeg',
+        'image/png',
+    ];
+
+    var FILE_TYPE_WHITE_LIST = ['pdf', 'doc', 'docx', 'html'];
+    var FILE_EXT_BLACK_LIST = ['exe', 'msi', 'app', 'dmg'];
```

Comment:
They are mock lists that are passed to the xblock js function as if they were passed from python. They don't have to match the list in python code as we don't test all file types in the list (only test one or a few of them to see if the logic is correct)